### PR TITLE
Fix unlike post

### PR DESCRIPTION
**Pull request by makaroni4**

### CODE REVIEW

File: app/controllers/account/likes_controller.rb [hunk 1]
```diff
@@ -4,9 +4,20 @@ class Account::LikesController < ApplicationController
   respond_to :json
   
   def create
-    @post = Post.find_by_param params[:post_id]
```
Comment (releu) [inline app/controllers/account/likes_controller.rb:7]: Does it  work with private posts?

File: app/models/user.rb [hunk 2]
```diff
@@ -166,7 +170,7 @@ def subscribe tag
   end
   
   def oauth_token
-    account_github.token
+    account_github.try(:token)
```
Comment (releu) [inline app/models/user.rb:173]: All accounts have a token.

File: lib/tasks/posts.rake [hunk 3]
```diff
@@ -2,10 +2,11 @@ namespace :posts do
   task update_page_views: :environment do
     google_analytics = GA.new
 
-    Post.limit(ENV['LIMIT'] || 50).each do |post|
+    Post.all.each do |post|
```
Comment (releu) [inline lib/tasks/posts.rake:5]: find_each

Comment (makaroni4) [inline lib/tasks/posts.rake:5]: Agree, for > 1000 posts all will be eager.

**Pull request closed without merging**